CHANGELOG.md

```diff
@@ -3,11 +3,116 @@
+- Updating formatter to not drop `rego.v1` and `future.keywords` imports for v1 by @johanfylling in [#7224](https://github.com/open-policy-agent/opa/pull/7224)
+- Update docs and server binding address per OPA 1.0 specs by @ashutosh-narkar & @charlieegan3 in [#7140](https://github.com/open-policy-agent/opa/pull/7140)
+- Renaming `--rego-v1` cmd flag to `--v0-v1` by @johanfylling in [#7225](https://github.com/open-policy-agent/opa/pull/7225)
+
+
+### Topdown and Rego
+- topdown: move context.Context cancellation check by @srenatus in [#7210](https://github.com/open-policy-agent/opa/pull/7210)
+- Provide a more useful error message when there are conflicting default rules by @tjons in [#7164](https://github.com/open-policy-agent/opa/pull/7164)
+- Fix flakes in `topdown/cache` by @evankanderson in [#7188](https://github.com/open-policy-agent/opa/pull/7188)
+- Add description to all built-in function args and return values by @anderseknert in [#7153](https://github.com/open-policy-agent/opa/pull/7153)
+- Built-in function `to_number` now rejects "Inf", "Infinity" and "NaN" values by @sikehish in [#7203](https://github.com/open-policy-agent/opa/pull/7203)
+- Update eval_cancel_error logic to separate context canceled, timeout errors by @mchitten in [#7202](https://github.com/open-policy-agent/opa/pull/7202)
+
+### Runtime, Tooling, SDK
+
+- Respect runtime rego-version in RESTful policy API by @johanfylling in [#7183](https://github.com/open-policy-agent/opa/pull/7183)
+- Debugger: allow YAML to be used as input by @anderseknert in [#7178](https://github.com/open-policy-agent/opa/pull/7178)
+- `opa build`: provide an option to preserve print statements (#7194) by @me-viper in [#7195](https://github.com/open-policy-agent/opa/pull/7195)
+- Fix improper formatter behavior when comprehension contains comment by @tjons in [#7169](https://github.com/open-policy-agent/opa/pull/7169)
+- runtime: send version report less often when OPA long-running by @srenatus in [#7211](https://github.com/open-policy-agent/opa/pull/7211)
+- `opa eval`: Return error if illegal arguments passed with `--unknowns` flag by @kd-labs in [#7149](https://github.com/open-policy-agent/opa/pull/7149)
+- Enable direct error handling for bundle plugin trigger method by @torwunder in [#7143](https://github.com/open-policy-agent/opa/pull/7143)
+
+### Docs, Website, Ecosystem
 
-OPA now requires at least Go 1.22 to build.
+- Add VodafoneZiggo as adopters by @Parsifal-M in [#7154](https://github.com/open-policy-agent/opa/pull/7154)
+- Add opa-java-wasm to docs by @andreaTP in [#7199](https://github.com/open-policy-agent/opa/pull/7199)
+
+### Dependency Updates
+
+- (build) golangci-lint: v1.59.1 -> v1.60.1 by @srenatus in [#7175](https://github.com/open-policy-agent/opa/pull/7175)
+- github.com/containerd/containerd: v1.7.23 -> v1.7.24
+- github.com/fsnotify/fsnotify: v1.7.0 -> v1.8.0
+- golang.org/x/net: v0.30.0 -> v0.33.0
```

Comment:
This one was important for a CVE raised by scanners. So let's ensure that the latest nightly run is green before we push this out.

Reply:
I must confess I'm a bit confused about how the security scanner operates 😅.
I re-ran the nightly job, it checked out commit `b8a1376b3`, which is after the `golang.org/x/net` bump at `c97b640e6`; but it still fails on the version check: https://github.com/open-policy-agent/opa/actions/runs/12429564134/job/34703306030.

Reply:
The post merge job has failed to push an edge image for a free days.

Reply:
nightly is ✅ 🎉

Reply:
Thank you